docs/styles/demo/icon-docs.less

```diff
@@ -1,13 +1,14 @@
-.icon-demo {
-  text-align: center;
+.icon-media {
+  background-color: @gray-0;
+  border: 1px solid @gray-400;
+  height: 100%;
 
-  & > p {
-    margin: 0;
-  }
-
-  hx-icon {
-    height: 2em;
-    width: 2em;
+  .media__icon {
+    align-items: center;
+    //color: @purple-500; // present as option in Bowling
```

Comment:
There are two potential color options for the icons.

## Option A
No change to icon color. Icon inherits text color.
![screen shot 2018-03-19 at 8 03 21 pm](https://user-images.githubusercontent.com/545605/37630104-9afc15ce-2bb0-11e8-89e7-65685b4db56e.png)


## Option B
Purple 500 icon. Provides a slight contrast between the icon and the description.
![screen shot 2018-03-19 at 8 03 03 pm](https://user-images.githubusercontent.com/545605/37630109-a05eef8c-2bb0-11e8-8e52-2868eb38a731.png)

Reply:
Per discussion in Bowling, we'll leave the color as default. Please remove comment.